x-pack/plugins/monitoring/public/application/hooks/use_title.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { get } from 'lodash';
+import { i18n } from '@kbn/i18n';
+import { useKibana } from '../../../../../../src/plugins/kibana_react/public';
+
+// TODO: verify that works for all pages
+export function useTitle(cluster: string, suffix: string) {
```

Comment:
This hook replaces the [title.js service](https://github.com/elastic/kibana/blob/master/x-pack/plugins/monitoring/public/services/title.js).